docs/Home.md

```diff
@@ -170,35 +170,34 @@ provided by the helm project for configuring helm/tiller on your GKE cluster.
   https://cloud.google.com/kubernetes-engine/docs/how-to/role-based-access-control)
 * [Configuring RBAC for GKE deployment](
   https://codeascraft.com/2018/06/05/deploying-to-google-kubernetes-engine/)
-  
+
 ## Usage
 
 ### Google Kubernetes Engine Build Step Configuration
 
 Each GKE Build Step configuration can point to a different GKE cluster. Follow the steps below to
 create one.
 
-#### GKE Build Step Parameters
+##### GKE Build Step Parameters
```

Comment:
Nit: Why did you change the header level here?

Reply:
It looked like a step in the build configuration to me, so I thought I'd try to have it distinguished from the other headers.

Reply:
Makes sense, thanks.